### DISCUSSION TITLE
Support for JEST 28?

### DISCUSSION THREAD

**Discussion by tim-rohrer** [typegoose/mongodb-memory-server#637] [Q&A] [answered]
Has anyone forced an update to Jest 28? Any problems?

I've got a project for which I'd like to update to Jest 28, but the dependency here is limited to the 27 series. Before I force it for this project, I wanted to check with some others to see if you're using Jest 28 with mongodb-memory-server successfully?

Thanks!

**Comment by hasezoey** [comment, marked as answer]
current, i had not tried to update yet (have not seen a reason to do it yet)

> but the dependency here is limited to the 27 series

from what i know, this project (all packages) does not have direct dependencies to jest, only devdependencies, though the examples are also only for jest 27

---

i have also read through the jest 28 changelog, but have not seen anything that would be breaking to how this package can be used

**Reply by tim-rohrer** [reply]
Thank you for the reply. Yes, as a devDep I get the risk is minimal, but I’d like to avoid tracking down gremlins, at least with this project. 

I’ve another project I’m working on where I may roll up to Jest 28 and see if there are any issues. Your last comment gave me the confidence to do that. Thank you.

**Reply by hasezoey** [reply]
i just tested myself on MMS with jest 28 and ts-jest 28, and it runs fine without problems
